packages/material-ui-lab/src/TimelineOppositeContent/timelineOppositeContentClasses.js

```diff
@@ -0,0 +1,14 @@
+import { generateUtilityClass, generateUtilityClasses } from '@material-ui/unstyled';
+
+export function getTimelineOppositeContentUtilityClass(slot) {
+  return generateUtilityClass('MuiTimelineOppositeContent', slot);
+}
+
+const timelineOppositeContentClasses = generateUtilityClasses('MuiTimelineOppositeContent', [
+  'root',
+  'alignLeft',
+  'alignRight',
+  'alignAlternate',
```

Comment:
I saw new classes added in #25781. Replicate this here as it also depends on the `align` props of the` Timeline` component that is passed by the context

Reply:
👍